apps/core/components/ItineraryCard/itineraryDetail/ItineraryDetail.js

```diff
@@ -14,12 +14,12 @@ type Props = {|
   +data: ?ItineraryType,
   +localizedPrice: string,
   +onClose: () => void,
-  +onBookPress: string => void,
+  +onBookPress: (token: ?string) => void,
```

Comment:
Why do you use `token` here? Why not just `?string`?

Reply:
Right, updated.